File: Modul-1/python/PRAK107-2310817110006-Adrian_Bintang_Saputera.py
a = float(input("Panjang sisi segitiga a: "))
b = float(input("Panjang sisi segitiga b: "))
c = float(input("Panjang sisi segitiga c: "))

keliling_tanah = a + b + c
harga_tanah_per_meter = 85000
biaya_yang_diperlukan = keliling_tanah * harga_tanah_per_meter

print("Diketahui :")
print(f"Panjang sisi segitiga berturut-turut adalah {a:.0f}, {b:.0f}, dan {c:.0f}")
print(f"Keliling Tanah Pak Dengklek adalah {keliling_tanah:.0f}")
print(f"Harga tanah Per Meter adalah {harga_tanah_per_meter}")
print("Jawaban :")
print(f"Biaya yang diperlukan Pak Dengklek adalah : Rp {biaya_yang_diperlukan:.0f}")

File: Modul-1/C/PRAK107-2310817110006-Adrian_Bintang_Saputera.c
#include <stdio.h>

int main()
{
    float a, b, c;
    printf("Panjang sisi segitiga a: ");
    scanf("%f", &a);
    printf("Panjang sisi segitiga b: ");
    scanf("%f", &b);
    printf("Panjang sisi segitiga c: ");
    scanf("%f", &c);

    float kelilingTanah = a + b + c;
    int hargaTanahPerMeter = 85000;
    int biayaYangDiperlukan = kelilingTanah * hargaTanahPerMeter;

    printf("Diketahui :\n");
    printf("Panjang sisi segitiga berturut-turut adalah %.0f, %.0f, dan %.0f\n", a, b, c);
    printf("Keliling Tanah Pak Dengklek adalah %.0f\n", kelilingTanah);
    printf("Harga tanah Per Meter adalah %d\n", hargaTanahPerMeter);
    printf("Jawaban :\n");
    printf("Biaya yang diperlukan Pak Dengklek adalah : Rp %d\n", biayaYangDiperlukan);

    return 0;
}